### DISCUSSION TITLE
Disabling the default menu

### DISCUSSION THREAD

**Discussion by astrogewgaw** [lowdefy/lowdefy#755] [Questions and Help] [answered]
<div align="justify">

If it cannot a find a menu in the `lowdefy.yaml` file, **Lowdefy** builds a default menu. While that is all well and good, it does the same thing even if I have a menu specified for each page, but do not have anything in the `lowdefy.yaml` file. Thus, I end up getting multiple menus in a single page 😅 (one that I built and another added by **Lowdefy**). Is there a way to stop **Lowdefy** from doing that?
 
</div>

**Comment by Gervwyk** [comment, marked as answer]
Hi @astrogewgaw 

Lowdefy generates a menu with `id: default` by default unless a menu of `id: default` is provided in the `lowdefy.yaml` to overwrite this default. See (the menu section on the schema page for a more detailed description and example.)[https://docs.lowdefy.com/lowdefy-schema] 

Thus you can easily provide an empty `default` menu to overwrite the generated default:
```yaml
lowdefy: LOWDEFY_VERSION
menus:
  - id: default
    links: []
```

As for the multiple menus on each page, how are you defining your menu on the page?

**Reply by astrogewgaw** [reply]
@Gervwyk Thanks for this 😁 ! I was getting multiple menus on a single page because one of them was the default menu. It is now working perfectly 👍🏾 !
